cli/tests/testdata/node/test.out

```diff
@@ -0,0 +1,176 @@
+escaped diagnostic ... cancelled (0ms)
+callback pass ... cancelled (0ms)
+callback fail ... cancelled (0ms)
+sync t is this in test ... cancelled (0ms)
+async t is this in test ... cancelled (0ms)
+callback t is this in test ... cancelled (0ms)
+callback also returns a Promise ... cancelled (0ms)
+callback throw ... cancelled (0ms)
+callback called twice ... cancelled (0ms)
+callback called twice in different ticks ... cancelled (0ms)
+callback called twice in future tick ... cancelled (0ms)
+callback async throw ... cancelled (0ms)
+callback async throw after done ... cancelled (0ms)
+custom inspect symbol fail ... cancelled (0ms)
+custom inspect symbol that throws fail ... cancelled (0ms)
+subtest sync throw fails ... cancelled (0ms)
+timed out async test ... cancelled (0ms)
+timed out callback test ... cancelled (0ms)
+large timeout async test is ok ... cancelled (0ms)
+large timeout callback test is ok ... cancelled (0ms)
+successful thenable ... cancelled (0ms)
+rejected thenable ... cancelled (0ms)
+unfinished test with uncaughtException ... cancelled (0ms)
+unfinished test with unhandledRejection ... cancelled (0ms)
+
+ ERRORS 
+
+sync fail todo => node:test:135:10
+error: Error: thrown from sync fail todo
+  throw new Error("thrown from sync fail todo");
+[WILDCARD]
+
+sync fail todo with message => node:test:135:10
+error: Error: thrown from sync fail todo with message
+  throw new Error("thrown from sync fail todo with message");
+[WILDCARD]
+
+sync throw fail => node:test:135:10
+error: Error: thrown from sync throw fail
+  throw new Error("thrown from sync throw fail");
+[WILDCARD]
+
+async throw fail => node:test:135:10
+error: Error: thrown from async throw fail
+  throw new Error("thrown from async throw fail");
+[WILDCARD]
+
+async skip fail => node:test:135:10
+error: Error: thrown from async throw fail
+  throw new Error("thrown from async throw fail");
+[WILDCARD]
+
+async assertion fail => node:test:135:10
+error: AssertionError: Values are not strictly equal:
+
+
+    [Diff] Actual / Expected
+
+
+-   true
++   false
+
+    at [WILDCARD]
+
+reject fail => node:test:135:10
+error: Error: rejected from reject fail
+  return Promise.reject(new Error("rejected from reject fail"));
+                        ^
+    at [WILDCARD]
+
+./node/test.js (uncaught error)
+error: Error: rejected from unhandled rejection fail
+  Promise.reject(new Error("rejected from unhandled rejection fail"));
+                 ^
+    at [WILDCARD]
+This error was not caught from a test and caused the test runner to fail on the referenced module.
+It most likely originated from a dangling promise, event/timeout handler or top-level code.
+
+ FAILURES 
+
+sync fail todo => node:test:135:10
+sync fail todo with message => node:test:135:10
+sync throw fail => node:test:135:10
+async throw fail => node:test:135:10
+async skip fail => node:test:135:10
+async assertion fail => node:test:135:10
+reject fail => node:test:135:10
```

Comment:
Perhaps we should wildcard these line numbers?

Reply:
I'd rather not - they show a problem that the current implementation doesn't point to proper place - all point to a wrapper function in `ext/node/polyfills/testing.ts` module. I think I know how to fix it in the implementation of `Deno.test` but I want to do it in a follow up.